progetto/ClientProject/src/client/Client.java

```diff
@@ -245,7 +244,7 @@ public boolean disconnect(){
 
     public boolean retrieveAccount(){
         try{
-            ResponseCode response=server_stub.retrieveAccount(username,plainPassword,skeleton,cookie);
+            ResponseCode response=server_stub.retrieveAccount(username,plainPassword,skeleton);
```

Comment:
Non ho capito questa modifica

Reply:
non deve fare la retrieve in base al cookie

Reply:
Il client una volta che chiude il programma dovrebbe salvarsi il cookie da qualche parte, è più logico ricavarlo dall'userName. Anche considerando che teoricamente i cookie sono temporanei e che dopo un tot di tempo dovrebbero diventare non più validi

Reply:
ma il cookie funzionava in questo modo
accountId<--->f(AES)<----->cookie
In questo modo era 0(1)

come fa ora a trovare l'accountId?

accountId sarebbe la posizione dove è salvato l'account nell'array degli account